.build-linux/publishViewsInSchema.ps1

```diff
@@ -132,8 +131,9 @@ ForEach-Object {
     $nuspec = '<?xml version="1.0"?><package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd"><metadata>'
     $nuspec += "<id>$packageId</id>"
     $nuspec += "<version>$version$metadata</version>"
-    $nuspec += "<authors>$machineName</authors>"
+    $nuspec += "<authors>$packageId</authors>"
```

Comment:
Why this change?

Reply:
The machine name makes little sense when we're working with action runners. Honestly it didn't make much sense when we were working with other build machines. The `packageId` seemed like a good enough replacement 🤷‍♂️ Maybe we should have a parameter to customize it, but I can always come back and do something like that if we really want.